.github/workflows/load-test.yml

```diff
@@ -3,6 +3,7 @@ on:
     branches:
       - main
     types: [labeled]
+  workflow_dispatch: {}
```

Comment:
I think this was inadvertently included from [another PR](https://github.com/hashicorp/consul/pull/13068)?

Reply:
thanks for catching this. Will remove this line in the update commit.